staging/src/k8s.io/client-go/util/flowcontrol/backoff_test.go

```diff
@@ -250,3 +250,34 @@ func TestBackoffWithJitter(t *testing.T) {
 
 	t.Logf("exponentially backed off jittered delays: %v", delays)
 }
+
+func TestAlternateHasExpiredFunc(t *testing.T) {
+	id := "_alternateHasExpiredFunc"
+	tc := testingclock.NewFakeClock(time.Now())
+	step := time.Second
+	maxDuration := 50 * step
+
+	b := NewFakeBackOff(step, maxDuration, tc)
+
+	// try several short (<maxduration) expirations that we can test with
+	cases := []time.Duration{1, 2, 4, 8}
```

Comment:
nit: it's hard for me to tell what these are testing. I'd prefer to use a normal table style, with a descriptive name. Maybe also set the func in the test case, so that you can have a test case that verifies the default expiration func.

Something like:
```go
type testCase struct {
  name string
  hasExpiredFunc func(time.Time, time.Time, time.Duration) bool
  nonExpiredTime time.Duration
  expiredTime time.Duration
}
```

Reply:
Fixed in https://github.com/kubernetes/kubernetes/commit/815001e983e700b053f236bcea49b794f20a65dc, it felt natural to collapse these all into one really when doing so, let me know if you agree with the cases